src/apps/core/views/monster-panel-text-danger.html

```diff
@@ -1,6 +1,6 @@
 <div class="monster-panel-text {{#if className}} {{className}}{{/if}}" data-type="danger">
 	<div class="monster-panel-text-header clearfix">
-		<div class="icon-wrapper"><i class="fa fa-exclamation-circle"></i></div>
+		<div class="icon-wrapper"><i class="fa fa-exclamation-circle icon"></i></div>
```

Comment:
2 things:

- try not using the `icon` class as it might inherit styling from bootstrap and telicon style sheets (I personally use `iconography` instead but you can use anything)
- wy not use the `warning-triangle` icon from telicon2?

Reply:
OK. About the `warning-triangle`, I did not use that one because it is the same for warning. But on a second thought, it does not matter a lot because it will have a different color.

Reply:
Maybe check with Josh before making that change. It does seem like telicon2 does not have an `error` specific icon.

Reply:
Will use `x--circle` as suggested by Josh.